### PR TITLE
DX copy-edit

### PR DESCRIPTION
**Pull request by jongio**
The PR is large, but most of the changes are tweaks are just copy-edit to clear up language.

### CODE REVIEW

File: pages/workshop/cs/01-getting-started.md [hunk 1]
```diff
@@ -14,11 +14,11 @@ permalink: /workshop/cs/nightlight/getting-started/
 *  Auto generated table of contents
 {:toc}
 
-## Preparing for the Windows 10 IoT Labs
-The labs in this series build on each other to enable you to prototype your own Internet of Things (IoT) devices. In this lab you will use the Microsoft .NET Framework to build an application for the Universal Windows Platform (UWP) that can run on any Windows 10 device, including a Raspberry Pi 2, MinnowBoard MAX, or DragonBoard 410c running Windows 10 IoT Core.
+## Preparing for the Windows 10 IoT - Connected Nightlight Workshop
+The labs in the Connected Nightlight Workshop build on each other to enable you to prototype your own Internet of Things (IoT) devices. You will use the Microsoft .NET Framework to build an application for the Universal Windows Platform (UWP) that can run on any Windows 10 device, including a Raspberry Pi 2, MinnowBoard MAX, or  a DragonBoard 410c running Windows 10 IoT Core.
```
Comment (rpedersen) [inline pages/workshop/cs/01-getting-started.md]: There is an extra space between the "or" and "a" before DragonBoard 410c

File: pages/workshop/cs/02-setup-rpi2.md [hunk 2]
```diff
@@ -61,16 +61,16 @@ Windows 10 IoT Core will boot on power-up. The first boot may take a few minutes
...
 2. Change the password if you would like.
 3. Change the name to something unique (remember, you are naming a Windows device - all normal naming rules and practices apply).
 4. Click __Accept__ to accept the changes and reboot the RPi2
 
-Your RPi2 will reboot and when it is back up and running it will have the new name you gave it, and the Administrator account will use the password you created. 
+Your Raspberry Pi 2 will reboot and when it is back up and running it will have the new name you gave it, and the Administrator account will use the password you created. 
 
 # Conclusion &amp; Next Steps
-In this lab you prepared your Raspberry Pi 2 for the following labs. Next you will do the IoT equivalent 'Hello, World!' - you will make an LED blink.
+In this section you prepared your Raspberry Pi 2 for the subsequent labs. Next you will do the IoT equivalent  of a 'Hello, World!' program: you will make an LED blink.
```
Comment (rpedersen) [inline pages/workshop/cs/02-setup-rpi2.md]: There is an extra space before "of a" prior to 'Hello, World!'

File: pages/workshop/thingy-4-windows/00-home.md [hunk 3]
```diff
@@ -76,27 +75,27 @@ In this lab, you will build a device with multiple sensors and actuators. The de
...
 
 ## Lab 06: Storing and Displaying IoT Data
-In this lab, you will build a data pipeline that captures the data coming into IoT Hub, processes it with Azure Stream Analytics, then routes it to downstream services. The first service you'll build will consume the data and present it through an Azure Web App where it is rendered as a real-time graph.
+In this lab, you will build a data pipeline that captures the data coming into an IoT Hub. The pipeline will then process the data with Azure Stream Analytics, and then route it to downstream services. You will also create one of those downstream services. Your service will consume the data and present it through an Azure Web App where it will be rendered as a real-time graph.
 
-<a class="radius button small" href="{{ site.url }}/workshop/thingy-4-windows/storing-displaying-data/">Estimated time to complete: 20-minutes. Go to  'Storing & Displaying IoT Data' ›</a>
+<a class="radius button small" href="{{ site.url }}/workshop/thingy-4-windows/storing-displaying-data/">Estimated time to complete: 30-minutes. Go to  'Lab 06: Storing & Displaying IoT Data' ›</a>
```
Comment (rpedersen) [inline pages/workshop/thingy-4-windows/00-home.md]: Extra space before 'Lab 06:

File: pages/workshop/thingy-4-windows/00-home.md [hunk 4]
```diff
@@ -76,27 +75,27 @@ In this lab, you will build a device with multiple sensors and actuators. The de
...
 
-## Lab 07: Lab: Sending Cloud-to-Device (C2D) Messages
+## Lab 07: Sending Cloud-to-Device (C2D) Messages
 In this lab, you will use the Azure Web App to control the ThingLabs Thingy&trade; remotely. The Web App will send messages to the Thingy via Azure, which the Thingy will receive, procees, and act on.
 
-<a class="radius button small" href="{{ site.url }}/workshop/thingy-4-windows/sending-c2d-messages/">Estimated time to complete: 20-minutes. Go to  'Sending Cloud to Device (C2D) Messages' ›</a>
+<a class="radius button small" href="{{ site.url }}/workshop/thingy-4-windows/sending-c2d-messages/">Estimated time to complete: 20-minutes. Go to  'Lab 07: Sending Cloud to Device (C2D) Messages' ›</a>
```
Comment (rpedersen) [inline pages/workshop/thingy-4-windows/00-home.md]: Extra space before 'Lab 07:

File: pages/workshop/thingy-4-windows/07-sending-c2d-messages.md [hunk 5]
```diff
@@ -1,6 +1,6 @@
 ---
 layout: "page-fullwidth"
-title: "Sending Cloud-to-Device (C2D) Messages"
+title: "Lab 97: Sending Cloud-to-Device (C2D) Messages"
```
Comment (rpedersen) [inline pages/workshop/thingy-4-windows/07-sending-c2d-messages.md]: hould be "Lab 7:"

**Review by rpedersen — CHANGES_REQUESTED** [review]
minor issues with extra spaces
One spot where lab 97 needs to be lab 7

**Comment by jongio** [comment]
Edits made.  Thanks for the review!

**Pull request merged**